### PR TITLE
UG: Updated command summary

### PR DESCRIPTION
**Pull request by chanyijuan**
* Improved markapplicant format and example in command summary

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -93,7 +94,7 @@ They are used first before entering adding to corresponding fields.
 * For example: `n/Benson Goh` - the `n/` flag is used to indicate that the subsequent input will be the `Applicant Name`.
 * For example: `jt/Project Manager` - the `jt/` flag is used to indicate that the subsequent input will be the `Job Title`.
 
-They are commonly used in every command. Details are given below
+They are commonly used in every command. Details are.g.iven below
```
Comment (seanleong339) [inline docs/UserGuide.md:97]: There is a typo here in given

File: docs/UserGuide.md [hunk 2]
```diff
@@ -134,28 +135,28 @@ They are commonly used in every command. Details are given below
 **:information_source: Notes about the command format:**<br>
 
 * Words in `UPPER_CASE` are the parameters to be supplied by the user.<br>
-  e.g. in `addapplicant n/NAME`, `NAME` is a parameter which can be used as `addapplicant n/John Doe`.
+  e.g.. in `addapplicant n/NAME`, `NAME` is a parameter which can be used as `addapplicant n/John Doe`.
```
Comment (seanleong339) [inline docs/UserGuide.md:138]: Double . Could be a typo

File: docs/UserGuide.md [hunk 3]
```diff
@@ -134,28 +135,28 @@ They are commonly used in every command. Details are given below
...
-  e.g `*n/[NAME] t/[TAG]` can be used as `n/John Doe t/Applicant` or `n/John Doe`.
+  e.g. `*n/[NAME] t/[TAG]` can be used as `n/John Doe t/Applicant` or `n/John Doe`.
 
 * Items with `…​` after them can be used multiple times including zero times.<br>
-  e.g. `[t/TAG]…​` can be used as ` ` (i.e. 0 times), `t/friend`, `t/friend t/family` etc.
+  e.g.. `[t/TAG]…​` can be used as ` ` (i.e. 0 times), `t/friend`, `t/friend t/family` etc.
 
 * Parameters can be in any order.<br>
-  e.g. if the command specifies `n/NAME p/PHONE_NUMBER`, `p/PHONE_NUMBER n/NAME` is also acceptable.
+  e.g.. if the command specifies `n/NAME p/PHONE_NUMBER`, `p/PHONE_NUMBER n/NAME` is also acceptable.
```
Comment (seanleong339) [inline docs/UserGuide.md:150]: Double . Could be a typo

File: docs/UserGuide.md [hunk 4]
```diff
@@ -134,28 +135,28 @@ They are commonly used in every command. Details are given below
...
 
 * If a parameter is expected only once in the command, but you specified it multiple times, only the last occurrence of
   the parameter will be taken.<br>
-  e.g. if you specify `p/12341234 p/56785678`, only `p/56785678` will be taken.
+  e.g.. if you specify `p/12341234 p/56785678`, only `p/56785678` will be taken.
```
Comment (seanleong339) [inline docs/UserGuide.md:154]: Double . Could be a typo

File: docs/UserGuide.md [hunk 5]
```diff
@@ -134,28 +135,28 @@ They are commonly used in every command. Details are given below
...
 
 * Extraneous parameters for commands that do not take in parameters 
   (such as `help`, `listapplicant`, `listjob`, `tabapplicant`, `tabjob`, `exit` and `clear`)
   will be ignored.<br>
-  e.g. if the command specifies `help 123`, it will be interpreted as `help`.
+  e.g.. if the command specifies `help 123`, it will be interpreted as `help`.
```
Comment (seanleong339) [inline docs/UserGuide.md:159]: Could be a typo. Double .

File: docs/UserGuide.md [hunk 6]
```diff
@@ -371,11 +372,11 @@ Format: `findapplicant KEYWORD…​`
 *flag/[Attribute]*
 
 `KEYWORD…​`: Search for an applicant name containing the `KEYWORD…​`.
-* The search is case-insensitive. e.g `alice` will match `Alice`
-* The order of the keywords does not matter. e.g. `Alice Goh` will match `Goh Alice`
-* Only full words will be matched e.g. `Ali` will not match `Alice`
+* The search is case-insensitive. e.g. `alice` will match `Alice`
+* The order of the keywords does not matter. e.g.. `Alice Goh` will match `Goh Alice`
```
Comment (seanleong339) [inline docs/UserGuide.md:376]: Could be a typo. double .

File: docs/UserGuide.md [hunk 7]
```diff
@@ -371,11 +372,11 @@ Format: `findapplicant KEYWORD…​`
...
+* Only full words will be matched e.g.. `Ali` will not match `Alice`
```
Comment (seanleong339) [inline docs/UserGuide.md:377]: Typo

File: docs/UserGuide.md [hunk 8]
```diff
@@ -371,11 +372,11 @@ Format: `findapplicant KEYWORD…​`
...
 * Applicants matching at least one keyword will be returned (i.e. `OR` search).
-  e.g. `Alice` will return `Alice Tan Mei Ling`, `Alice Goh Ming Yue`
+  e.g.. `Alice` will return `Alice Tan Mei Ling`, `Alice Goh Ming Yue`
```
Comment (seanleong339) [inline docs/UserGuide.md:379]: Typo

File: docs/UserGuide.md [hunk 9]
```diff
@@ -491,7 +492,7 @@ indicates that the salary is not a range, but rather a fixed amount.
 
 Examples:
 
-`editjob 1 jt/Software Engineer UI c/Desginer Club q/Degree in Computer Science a/123 Block 3 Designer Road pos/ft 
+`editjob 1 jt/Software Engineer UI c/De.g.iner Club q/Degree in Computer Science a/123 Block 3 Designer Road pos/ft
```
Comment (seanleong339) [inline docs/UserGuide.md:495]: Typo De.g.iner

File: docs/UserGuide.md [hunk 10]
```diff
@@ -567,16 +568,16 @@ Format (search via job ID): `findjob id/[ID]`
 ***flag/[Attribute]***
 
 `jt/[KEYWORD] KEYWORD`: Search for job title containing the `[KEYWORD]`. 
-* The search is case-insensitive. e.g `engineer` will match `Engineer`
-* The order of the keywords does not matter. e.g. `Software Engineer` will match `Engineer Software`
-* Only full words will be matched e.g. `Software` will not match `Soft`
+* The search is case-insensitive. e.g. `e.g.ineer` will match `Engineer`
```
Comment (seanleong339) [inline docs/UserGuide.md:571]: Typo e.g.ineer

File: docs/UserGuide.md [hunk 11]
```diff
@@ -567,16 +568,16 @@ Format (search via job ID): `findjob id/[ID]`
...
+* Only full words will be matched e.g.. `Software` will not match `Soft`
 * Jobs matching at least one keyword will be returned (i.e. `OR` search).
-  e.g. `Engineer` will return `Software Engineer`, `Chemical Engineer`
+  e.g.. `Engineer` will return `Software Engineer`, `Chemical Engineer`
 
 `id/[ID]` : Search for job with matching `[ID]`.
 
 Examples:
-* `findjob jt/Engineer` returns `software engineer` and `Mech Engineer`
+* `findjob jt/Engineer` returns `software e.g.ineer` and `Mech Engineer`
```
Comment (seanleong339) [inline docs/UserGuide.md:580]: Typo in e.g.ineer

**Review by seanleong339 — CHANGES_REQUESTED** [review]
Typos in the docs, not sure if intentional

**Review by Justinnje — APPROVED** [review]
LGTM

**Review by seanleong339 — APPROVED** [review]
LGTM

**Pull request merged**